test/unit/merge.js

```diff
@@ -267,20 +267,17 @@ describe('Merge', function () {
         this.merge.addFileAsync('local', _.cloneDeep(offUpdate))
         )
 
+        const doc = builders
+          .metafile(initialFile)
+          .data('off update')
+          .updatedAt(offUpdate.updated_at)
+          .sides({ local: 2 })
+          .build()
+        delete doc._rev // Side effects don't include doc's _rev
+        delete doc.remote // FIXME: stop mixing no remote attribute and undefined remote attribute
```

Comment:
I suppose we can't use `BaseMetadataBuilders#noRemote()` for this, since it sets `doc.remote` to `undefined`... :sob: